evm/vm/flavors/homestead/blocks.py

```diff
@@ -8,3 +14,8 @@
 
 class HomesteadBlock(FrontierBlock):
     transaction_class = HomesteadTransaction
+    fields = [
+        ('header', BlockHeader),
+        ('transactions', CountableList(HomesteadTransaction)),
+        ('uncles', CountableList(BlockHeader))
+    ]
```

Comment:
Side note:

It would be nice if rlp objects supported *proper* inheritance in much the same way as django models work.  We'd need a metaclass to inherit from and a `Field` class of some sort for each field.  Would be a better/nicer interface and would allow more fine-grained subclassing